mock-up/scss/titles.scss

```diff
@@ -0,0 +1,12 @@
+.title {
+  align-items: center;
+  display: flex;
+  padding: 0 10em;
```

Comment:
Use absolute units. Here more [info](https://css-tricks.com/the-lengths-of-css/).